.gitmodules

```diff
@@ -15,4 +15,4 @@
 	url = https://github.com/sociomantic-tsunami/turtle.git
 [submodule "beaver"]
 	path = submodules/beaver
-	url = https://github.com/sociomantic-tsunami/beaver.git
+	url = https://github.com/leandro-lucarella-sociomantic/beaver.git
```

Comment:
?

Reply:
Just for testing the unreleased beaver that adds support for this (that's why it's WIP and blocked and the commit message says I need to change that before going final :D)